examples/lightrag_nvidia_demo.py

```
@@ -0,0 +1,159 @@
+    os.mkdir(WORKING_DIR)
+
+#some method to use your API key (choose one)
+# NVIDIA_OPENAI_API_KEY = os.getenv("NVIDIA_OPENAI_API_KEY")
+NVIDIA_OPENAI_API_KEY = "nvapi-xxxx" #your api key
+
+# using pre-defined function for nvidia LLM API. OpenAI compatible
+# llm_model_func = nvidia_openai_complete
+
+#If you trying to make custom llm_model_func to use llm model on NVIDIA API like other example:
+async def llm_model_func(
+    prompt, system_prompt=None, history_messages=[], keyword_extraction=False, **kwargs
```

Ruff (0.8.0)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)